### ISSUE TITLE
Dragging a URL from Chrome isn't working

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#1425]
This works from Firefox and Safari, and I'm pretty sure this used to work from Chrome. We should look at the drop data to see what's coming in.

**Comment by dstillman** [comment]
This appears to be a bug in Chrome.

Dragging from the address bar:

```
{
    "dataType": "text/x-moz-url"
    "data": [
        "0": "https://www.zotero.org/"
    ]
    "dropEffect": "move"
}
```

Dragging from the page:

```
{
    "dataType": ""
    "data": []
    "dropEffect": "move"
}
```

**Comment by AbeJellinek** [comment]
No longer relevant after #4649
